frontends/PyCDE/src/pycde/module.py

```diff
@@ -248,11 +288,20 @@ def module(func_or_class):
   parameterized module."""
   if inspect.isclass(func_or_class):
     # If it's just a module class, we should wrap it immediately
-    return _module_base(func_or_class, None)
+    mod = _module_base(func_or_class,
```

Comment:
This is actually leftover code from a partial implementation where `mod` was being added the callback attributes instead of carried into the function call. I'll revert to the prior implementation.